src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java

```diff
@@ -77,7 +93,23 @@ public void verifySize(Dimension expectedSize) {
     );
   }
 
-  public void closeInterstitial() {
+  public void closeInterstitial() throws InterruptedException {
+
+    boolean isMercury = false;
+    try {
+      isMercury = interstitialCloseButtonMercury.isDisplayed();
+    } catch (NoSuchElementException e) { /* ignore */ }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)